types/sanctuary/index.d.ts

```diff
@@ -72,33 +79,22 @@ interface Extend<A> extends Functor<A> {}
 interface Comonad<A> extends Extend<A> {}
 interface Contravariant<A> {}
 
-interface ListToMaybeList {
-  (xs: string): Maybe<string>;
-  <A>(xs: ReadonlyArray<A>): Maybe<A[]>;
-}
-
 interface MatchObj {
   match: string;
   groups: ReadonlyArray<Maybe<string>>;
 }
 
 declare namespace Sanctuary {
   interface Static {
-    Maybe: TypeRep;
-    Nothing: Maybe<any>;
-    Just<A>(x: A): Maybe<A>;
-    Either: TypeRep;
-    Left<A>(x: A): Either<A, any>;
-    Right<A>(x: A): Either<any, A>;
     //  Classify
     type(x: any): {
       namespace: Maybe<string>
       name: string
       version: NonNegativeInteger
     };
-    is(typeRep: TypeRep): (x: any) => boolean;
+    is(tk: TK): (tk: TK) => TK;
     //  Showable
-    toString(x: any): string;
+    show(tk: TK): TK;
```

Comment:
I think it should be done with an interface (making it possible to work with custom types), something like:
```
interface Showable {
  '@@show': () => string;
}

show(s: Showable | number | string | anyOtherType)
```

Reply:
Good idea!